app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result_list = []
+    for persons in people:
+        result_list.append(Person(persons.get("name"), persons.get("age")))
+    for add_person in people:
+        if add_person.get("wife"):
+            Person.people[add_person.get("name")].wife = Person.people[
+                add_person.get("wife")
```

Comment:
.get in this case is redundant because we already know that name and wife exists
Fix in husband also
```suggestion
            Person.people[add_person["name"]].wife = Person.people[add_person["wife"]]
```